Tools/flatpak/flatpakutils.py

```diff
@@ -817,7 +817,12 @@ def run_in_sandbox(self, *args, **kwargs):
 
             if args[0] == "bash":
                 args.extend(['--noprofile', '--norc', '-i'])
-                sandbox_environment["PS1"] = f"[📦🌐🐱 $FLATPAK_ID {self.platform}@{self.build_type} \\W]\\$ "
+                sandbox_environment["PS1"] = f"[$FLATPAK_ID {self.platform}@{self.build_type} \\W]\\$ "
+                try:
+                    if 'en_US.utf8' in subprocess.check_output(['locale', '-a']).decode("utf8").split():
+                        sandbox_environment["PS1"] = f"[📦🌐🐱 $FLATPAK_ID {self.platform}@{self.build_type} \\W]\\$ "
+                except:
```

Comment:
A blank except is bad form in general, it catches everything like hitting ctrl+c.

You probably want except subprocess.CalledProcessError:

Reply:
It was `FileNotFoundError`.